docs/CHANGELOG.asciidoc

```diff
@@ -30,6 +30,223 @@ This section summarizes the changes in each release.
+* [ML] Data Frames: Transform list refresh button. {pull}40129[#40129]
+* [Fixes #38776] [ML] Data Frames: Adds support for transform description {pull}40153[#40153]
+Management::
+* [Fixes #13804, #17894, #17956, #17969, #17978, #18008, #18008, #18031, #18047, #18052, #18151, #18191, #18245, #18246, #18256, #18268, #18271, #18296, #18353, #18506, #18508, #18532, #18536, #18794, #22615, #29167, #33207, #33917, #34932, #39828] EUIfy Watcher {pull}35301[#35301]
+* Separate Console History panel from editor with a drop-shadow, and add a Close button {pull}39972[#39972]
+* [Fixes #39985] Accept triple quoted input in Search Profiler. {pull}39984[#39984]
+* [SR] Snapshot and Restore UI {pull}39193[#39193]
+* EUIfy Console - partially de-angularize and move custom views out of top_nav {pull}39341[#39341]
+* Console update autocomplete definitions {pull}39508[#39508]
+Monitoring::
+* Removed dead code {pull}39545[#39545]
+* Issue 5429: Added a shortcut to ML jobs {pull}39147[#39147]
+Observability::
+* [Uptime] Summary based monitors Page {pull}39751[#39751]
+* [Fixes #38786, #57711] [Uptime] Add monitor summaries {pull}38895[#38895]
+* [Fixes #46] [Uptime] [Bug] Add error callout for uptime filter bar {pull}38140[#38140]
+* [Uptime] Add telemetry collector {pull}34437[#34437]
+* [Uptime] Add link for logs host integration {pull}38010[#38010]
+* [Fixes #40589] [Uptime] Add link to Uptime documentation from header help menu {pull}40645[#40645]
+Platform::
+* Query Filter \ Filter Manager: de-angularize and move to data plugin {pull}37311[#37311]
+* Introduce setting to disable batching in courier {pull}39470[#39470]
+Reporting::
+* Check if CSV cells (including headers) start with known formula characters {pull}37930[#37930]
+SIEM::
+* Kpi on host details page {pull}39681[#39681]
+* Adds Machine Learning table anomalies, a pop over UI for anomalies, and machine learning details score  {pull}39483[#39483]
+* Adds Machine Learning Permissions to hide or show UI elements based on the permissions {pull}40119[#40119]
+* Adds Machine Learning Anomaly Table to Host Details and Network Details {pull}40223[#40223]
+* Adds conditional linking within the application for machine learning jobs {pull}40547[#40547]
+* Adds ML Integration for Installing &amp; Starting/Stopping embedded ML jobs {pull}39766[#39766]
+* Siem inspect query {pull}39980[#39980]
+* Enhance inspect modal  {pull}40530[#40530]
+Security::
+* [Fixes #38563] Security - Allow roles to be cloned {pull}39303[#39303]
+Visualize::
+* Add dateHistogramInterval utility {pull}39091[#39091]
+
+
+[float]
+[[bug-7.3.0]]
+=== Bug fixes
+APM::
+* [Fixes #34697]  Distinguish between loading state and empty state {pull}40651[#40651]
+* [Fixes elastic/kibana#40477] [APM Tutorial] Fix APM Server link on Cloud {pull}40711[#40711]
+Canvas::
+* [Fixes #41296]  Apply global CSS to export workpad view {pull}41298[#41298]
+* [Fixes #39852]  Fix: native HTML5 drag&amp;drop shouldn't kick in {pull}39861[#39861]
+Code::
```

Comment:
@jinmu03 Should we be including Code PRs?

Reply:
Since Code was never released, I don't think it makes sense to include bug fixes.